posts/Davide/2018-05-25-hibernate-ogm-5-4-Beta1-released.adoc

```diff
@@ -0,0 +1,83 @@
+= "Hibernate OGM 5.4.0.Beta1 release"
+Davide D'Alto
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+http://hibernate.org/ogm/releases/5.4/#get-it[Hibernate OGM 5.4.0.Beta1] has been released!
+
+The main differences from the previous release is the upgrade to Hibernate ORM 5.3.0.Final, 
+the support for Infinispan Remote native queries and the use of cluster counters
+for local caches when generating sequences with Infinispan Embedded.
+
+All the changes are described in detail in the
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=31632[release notes].
+
+== Hibernate ORM 5.3
+
+You can find all the details about it in the
+http://in.relation.to/2018/05/14/hibernate-orm-530-final-release/[Hibernate ORM 5.3.0.Final release post].
+The main things I will mention here is that it comes with support for JPA 2.2.
+
+== Infinispan Remote queries
+
+You can execute queries in Infinispan using the query language called
+https://blog.infinispan.org/2016/12/meet-ickle.html[Ickle]. This latest version of Hibernate OGM
+will convert the JPQL queries into an equivalent in Ickle. You can also run native query directly.
+
+Here is an example of a native query execution:
+
+====
+[source, JAVA]
+----
+String ickleQuery = "from HibernateOGMGenerated.Registry where level > 3 order by start"
+List result = session.createNativeQuery( ickleQuery )
+				.addEntity( Employee.class )
+				.list();
+----
+====
+
+With Ickle you can also write queries using the Lucene syntax, as long as the field is
```

Comment:
Maybe we should use Ickle syntax instead of Lucene syntax. I think @Sanne is going to fix it.